Assets/_Functionalities/Wms/Scripts/WMSLayerGameObject.cs

```diff
@@ -39,18 +49,27 @@ protected override void Awake()
         {
             base.Awake();
             wmsProjectionLayer = GetComponent<WMSTileDataLayer>();
+
+            credentialHandler = GetComponent<ICredentialHandler>();
+            credentialHandler.OnAuthorizationHandled.AddListener(HandleCredentials);
+
+            //we need to resolve the listener to the datatypechain because this is a prefab and it doesnt know about whats present in the scene
+            DataTypeChain chain = FindObjectOfType<DataTypeChain>();
+            if(chain != null) 
```

Comment:
Deze snap ik niet - als er een LayerGameObject is, dan heb je niks meer met de chain te maken. Ik vraag mij ook af wat er nu gebeurd als je een project opent met meerdere WMS lagen; want die gaat dan ook aan de chain binden?